docs/src/lib/indexing.md

```diff
@@ -100,9 +169,21 @@ In such an operation `AbstractDataFrame` is considered as two-dimensional and `D
     The rule above means that, similar to single-dimensional objects in Base (e.g. vectors),
     `DataFrameRow` is considered to be column-oriented.
 
-If column indexing using `Symbol` names is performed the order of columns in the operation is specified
+Additional rules:
```

Comment:
This list is quite repetitive. How about listing the syntaxes which are in-place and those which aren't instead?

Reply:
I agree, but please leave it as is for now. I need it to write proper tests (and someone later checking the tests will need it to make sure that we have proper test coverage).

I propose to shorten the list after we are done with implementation and review of tests.